cookies/resources/set-cookie.py

```diff
@@ -20,10 +20,10 @@ def main(request, response):
     < Date: Tue, 04 Oct 2016 18:16:06 GMT
     < Content-Length: 80
     """
-    params = urlparse.parse_qs(request.url_parts.query)
+    params = parse_qs(request.url_parts.query)
     headers = [
-        ("Content-Type", "application/json"),
-        ("Set-Cookie", "{name[0]}=1; Path={path[0]}; Expires=Wed, 09 Jun 2021 10:18:14 GMT".format(**params))
+        (b"Content-Type", b"application/json"),
+        (b"Set-Cookie", "{name[0]}=1; Path={path[0]}; Expires=Wed, 09 Jun 2021 10:18:14 GMT".format(**params).encode("utf-8"))
```

Comment:
I'm not sure if this is supposed to be `utf-8`.

After #23284 , this whole handler could be refactored to use `request.GET["name"]` and `request.GET["path"]`, both of which are already binary, instead of parsing the query again.